### DISCUSSION TITLE
`language={english}`, not `language={en}`

### DISCUSSION THREAD

**Discussion by ryofurue** [retorquere/zotero-better-bibtex#2908] [Ideas]
My Zotero database has a lot of instances of `language="EN"`.  The Zotero webbrowser plugin inserts them.

Then, I need to translate Zotero's `language` to bibtex's `language` so that `biblatex` prints the name of the language if it's not English; for example, I need `biblatex` to print "in Chinese" in the reference list.  So, I select the BBT's switch to translate `language` to `language`.

But, currently, BBT translates `language="EN"` to `language={EN}`, which doesn't work.  It needs to be `language={english}`.

So, does it make sense to translate "EN" to "english" if the target is the `language` field?

**Comment by retorquere** [comment]
Open an issue, submit a debug log, and I'll have a look.

**Reply by ryofurue** [reply]
Sure.  I've just copied my post above and added a debug log there.
